src/pages/index.vue

```diff
@@ -57,7 +57,9 @@
             <PVColumn field="password" header="PASSWORD">
               <template #body="{ data }">
                 <div v-element-hover="() => (hasCopiedPassword = false)" class="flex flex-row items-center group/password">
-                  <span class="font-mono text-password-color text-[20px] mr-[8px]">{{ data.password }}</span>
+                  <span class="font-mono text-password-color text-[20px] mr-[8px]">
+                    <TextHighlighter v-model="input" :enable="findByPassword" :text-to-highlight="data.password" />
```

Comment:
```suggestion
                    <TextHighlighter v-model="debouncedInput" :enable="findByPassword" :text-to-highlight="data.password" />
```

Use the debounced input